CHANGES.md

```diff
@@ -2,15 +2,17 @@
 
 #### Bug fixes
 - Disable browser history on Chrome iOS ([#146])
-- Do not convert same-path PUSH to REPLACE if the hash has changed ([#167]) 
+- Do not convert same-path PUSH to REPLACE if the hash has changed ([#167])
 
 #### Other
 - Add ES2015 module build ([#152])
+- Replace invariant by warning in createMemoryHistory ([#170])
```

Comment:
Let's call this a bugfix (:

"Don't throw in memory history when out of history entries"